conformance/1.0/Makefile

```diff
@@ -0,0 +1,27 @@
+TEST_PROFILE ?= kf-conformance-test
+KUBEFLOW_NAMESPACE ?= kubeflow
+
```

Comment:
Should this folder be `conformance/1.5` to align with current kubeflow version?

Reply:
Good point. Will do it in the next commit.

Reply:
Fixed.